integ/scripts/bash/tear-down.sh

```diff
@@ -49,7 +49,7 @@ for COMPONENT in **/cdk.json; do
     fi
 done
 
-run_aws_interaction_hook
+run_aws_interaction_hook "$(timestamp)"
```

Comment:
Based on the code statement after this one, we can probably log this as being for infrastructure:

```suggestion
run_aws_interaction_hook "$(timestamp) [infrastructure]"
```

what do you think?

Reply:
Yep that makes sense, updated.